services/common/type/waiverTemporaryExtension.js

```diff
@@ -5,7 +5,7 @@ export const waiverTemporaryExtension = {
   componentType: "waiverextension",
   typeLabel: "Waiver Extension",
   idLabel: "Temporary Extension Request Number",
-  idRegex: "^[A-Z]{2}[-][0-9]{4,5}[.]R[0-9]{2}[.]TE[0-9]{2}$",
+  idRegex: "^[A-Z]{2}[-][0-9]{2,5}[.]R[0-9]{2}[.]TE[0-9]{2}$",
```

Comment:
Heh... that change to the Regex is because I made the mistake of looking at the PROD data to find examples of Approved Initial Waivers that are SEA Tool only and discovered that Maryland's current waivers are formatted as MD-##.R## with no amendment number and only two digits in the family portion.
We have the requirement to allow parent matches in whatever format they are, but then the prefilled ID isn't viable because it doesn't follow the ID format.  I updated the Regex to allow from 2-5 digits really just to see what would happen and missed changing it back in my cleanup... which I'm conflicted on...